docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -35,16 +34,25 @@ Ensure your Golang application exposes pprof endpoints.
     import _ "github.com/grafana/pyroscope-go/godeltaprof/http/pprof"
     ```
 
-### Install Grafana Agent
+### Install the collector
 
 [//]: # (TODO&#40;korniltsev&#41; What should go here?)
 
-This procedure uses Grafana Agent to send data to Pyroscope using an example River configuration file for the Grafana Agent in Flow mode.
-Make sure you have [Grafana Agent in Flow mode](/docs/agent/latest/flow/setup/install/) installed.
+This procedure can be used with either Grafana Alloy or Grafana Agent collector.
+The collector sends data to Pyroscope using an example River configuration file.
```

Comment:
This is only true for Agent.  Alloy uses the `Alloy configuration syntax` or  `Alloy configuration file`.

Reply:
What should we stay instead here?

Reply:
How is this?
```suggestion
You can use the sample collector configuration file to send data to Pyroscope.
```

Reply:
Maybe just drop `River`... and say `The collector sends data to Pyroscope using an example configuration file.`